src/foam/u2/ActionView.js

```diff
@@ -114,11 +121,10 @@ foam.CLASS({
       } else if (this.iconFontName) {
         this.nodeName = 'i';
         this.cssClass(this.action.name);
-        this.cssClass(this.myClass(this.iconFontClass));
         this.cssClass(this.iconFontClass); // required by font package
         this.style({'font-family': this.iconFontFamily});
         this.add(this.iconFontName);
-        this.setAttribute('title', this.action.name); // hover text
+        this.setAttribute('title', this.action.speechLabel); // hover text
```

Comment:
This is okay for now, but we probably want to distinguish between tooltip and speechlabel in the future.